CHANGELOG

```diff
@@ -11,8 +11,15 @@ The rules for CHANGELOG file:
 
 .. inclusion-marker-changelog-start
 
-0.3.0 (XXXX/XX/XX)
+0.2.1 (XXXX/XX/XX)
```

Comment:
Keep the version `0.3.0`. 

```suggestion
0.3.0 (XXXX/XX/XX)
```

In the `pyproject.toml` we also use version 0.3.0. And we will decide once we relase if we do a major/minor or patch release.

Reply:
This looks like not done?